### PR TITLE
lightning: update get started doc

### PR DESCRIPTION
**Pull request by ran-huang**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: get-started-with-tidb-lightning.md [hunk 1]
```diff
@@ -1,54 +1,61 @@
...
+你可以使用 [`dumpling`](/dumpling-overview.md) 从 MySQL 导出数据。
 
-{{< copyable "shell-regular" >}}
+1. 安装 Dumpling：
 
-```sh
-tiup dumpling -h 127.0.0.1 -P 3306 -u root -t 16 -F 256MB -B test -f 'test.t[12]' -o /data/my_database/
-```
+    ```shell
+    curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh && \
```
Comment (dsdashun) [inline get-started-with-tidb-lightning.md:21]: This command is to install TiUP, right ?  
Maybe you can make it as a separate step to verify whether TiUP is installed.  If not , use `curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh` to install the TiUP

File: get-started-with-tidb-lightning.md [hunk 2]
```diff
@@ -79,16 +86,14 @@ TiDB Lightning 的安装包位于 TiDB 离线工具包中。下载方式，请
     pd-addr = "172.16.31.3:2379"
```
Comment (dsdashun) [inline get-started-with-tidb-lightning.md:86]: why the tidb host and the PD address is a strange IP address here ?  If it is a hands-on tutorial,  the TiDB is by default installed on 127.0.0.1 , right ?

Comment (ran-huang) [reply get-started-with-tidb-lightning.md:86]: I'm not sure why the config file was set as such. Can you take a look at other parameters in the config and see if they are wrong as well?

File: get-started-with-tidb-lightning.md (hunk 1 above)
Comment (ran-huang) [reply get-started-with-tidb-lightning.md]: Fixed in [66194be](https://github.com/pingcap/docs-cn/pull/12059/commits/66194bef4b858dbf019518aad9d8dc51756742f6)

**Comment by qiancai** [comment]
Removed the **needs-cherry-pick-release-6.3** label because the v6.3 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.3 and will no longer receive new updates.

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-6.4 label because the v6.4 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.4 and will no longer receive new updates.

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-6.6 label because the v6.6 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.6 and will no longer receive new updates.

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-7.0 label because the v7.0 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v7.0 and will no longer receive new updates.

**Comment by qiancai** [comment]
Removed the `needs-cherry-pick-release-7.2` label because the v7.2 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v7.2 and will no longer receive new updates.

File: get-started-with-tidb-lightning.md [hunk 3]
```diff
@@ -1,54 +1,66 @@
...
-- `-B test`：从 `test` 数据库导出。
-- `-f test.t[12]`：只导出 `test.t1` 和 `test.t2` 这两个表。
-- `-t 16`：使用 16 个线程导出数据。
-- `-F 256MB`：将每张表切分成多个文件，每个文件大小约为 256 MB。
+    ```shell
+    tiup install dumpling
+    ```
 
-这样全量备份数据就导出到了 `/data/my_database` 目录中。
+3. 从 MySQL 导出数据：
```
Comment (Frank945946) [inline get-started-with-tidb-lightning.md:30]: 导出数据的内容可以参考这个章节https://docs.pingcap.com/zh/tidb/stable/dumpling-overview#%E5%AF%BC%E5%87%BA%E4%B8%BA-sql-%E6%96%87%E4%BB%B6

File: get-started-with-tidb-lightning.md [hunk 4]
```diff
@@ -1,54 +1,66 @@
...
-3. 配置 `tidb-lightning.toml`。
+运行如下命令，安装 TiDB Lightning 最新版本：
 
-    ```
+```shell
+tiup install tidb-lightning
+```
+
+## 第 4 步：启动 TiDB Lightning
+
```
Comment (Frank945946) [inline get-started-with-tidb-lightning.md:60]: https://docs.pingcap.com/zh/tidb/stable/migrate-large-mysql-to-tidb#%E7%AC%AC-2-%E6%AD%A5%E5%AF%BC%E5%85%A5%E5%85%A8%E9%87%8F%E6%95%B0%E6%8D%AE%E5%88%B0-tidb
Lightning 导入数据可以参考这个章节

**Review by Frank945946 — APPROVED** [review]

File: get-started-with-tidb-lightning.md [hunk 5]
```diff
@@ -1,54 +1,70 @@
...
 
 > **警告：**
 >
 > 本教程中的部署方法只适用于测试及功能体验，并不适用于生产或开发环境。
 
-## 准备全量备份数据
+## 第 1 步：准备全量备份数据
 
-我们使用 [`dumpling`](/dumpling-overview.md) 从 MySQL 导出数据，如下：
+你可以使用 [`dumpling`](/dumpling-overview.md) 从 MySQL 导出数据。
```
Comment (Oreoxmt) [inline get-started-with-tidb-lightning.md:16]:
```suggestion
你可以使用 [Dumpling](/dumpling-overview.md) 从 MySQL 导出数据。
```

File: get-started-with-tidb-lightning.md [hunk 6]
```diff
@@ -1,54 +1,70 @@
...
+    ```
 
-### 第 1 步：部署 TiDB 集群
+    其中：
 
-在开始数据导入之前，需先部署一套要进行导入的 TiDB 集群。本教程以 TiDB v5.4.0 版本为例，具体部署方法可参考[使用 TiUP 部署 TiDB 集群](/production-deployment-using-tiup.md)。
+    - `-B test`：从 `test` 数据库导出。
+    - `-f test.t[12]`：只导出 `test.t1` 和 `test.t2` 这两个表。
+    - `-t 16`：使用 16 个线程导出数据。
+    - `-F 256MB`：将每张表切分成多个文件，每个文件大小约为 256 MB。
```
Comment (Oreoxmt) [inline get-started-with-tidb-lightning.md:41]:
```suggestion
    - `-t 16`：使用 16 个线程导出数据。
    - `-F 256MB`：将每张表切分成多个文件，每个文件大小约为 256 MB。
    - `-B test`：从 `test` 数据库导出。
    - `-f 'test.t[12]'`：只导出 `test.t1` 和 `test.t2` 这两个表。
```

File: get-started-with-tidb-lightning.md [hunk 7]
```diff
@@ -1,54 +1,70 @@
...
 
-### 第 2 步：下载 TiDB Lightning 安装包
+    这样全量备份数据就导出到了 `/data/my_database` 目录中。
```
Comment (Oreoxmt) [inline get-started-with-tidb-lightning.md:43]:
```suggestion
    导出的全量备份数据将保存在 `/data/my_database` 目录中。
```

File: get-started-with-tidb-lightning.md [hunk 8]
```diff
@@ -1,54 +1,70 @@
...
 
-TiDB Lightning 的安装包位于 TiDB 离线工具包中。下载方式，请参考 [TiDB 工具下载](/download-ecosystem-tools.md)。
+## 第 2 步：部署 TiDB 集群
 
-### 第 3 步：启动 `tidb-lightning`
+在开始导入数据之前，你需要先部署一套要进行导入的 TiDB 集群。如果你已经有 TiDB 集群，可以跳过这一步。
```
Comment (Oreoxmt) [inline get-started-with-tidb-lightning.md:47]:
```suggestion
在开始导入数据之前，你需要先部署一个要进行导入的 TiDB 集群。如果你已经有 TiDB 集群，可以跳过这一步。
```

File: get-started-with-tidb-lightning.md [hunk 9]
```diff
@@ -1,54 +1,70 @@
...
 
-1. 将安装包里的 `bin/tidb-lightning` 及 `bin/tidb-lightning-ctl` 上传至部署 TiDB Lightning 的服务器。
+关于部署 TiDB 集群的步骤，请参考 [TiDB 数据库快速上手指南](/quick-start-with-tidb.md)。
 
-2. 将数据源也上传到同样的服务器。
+## 第 3 步：安装 TiDB Lightning
 
-3. 配置 `tidb-lightning.toml`。
+运行如下命令，安装 TiDB Lightning 最新版本：
```
Comment (Oreoxmt) [inline get-started-with-tidb-lightning.md:53]:
```suggestion
运行如下命令，安装 TiDB Lightning 的最新版本：
```

File: get-started-with-tidb-lightning.md [hunk 10]
```diff
@@ -1,54 +1,70 @@
...
+tiup install tidb-lightning
+```
+
+## 第 4 步：启动 TiDB Lightning
+
+> **注意：**
+>
+> 本节的导入方法只适用于测试及功能体验，生产环境请参考[从大数据量 MySQL 迁移数据到 TiDB](/migrate-large-mysql-to-tidb.md#第-2-步导入全量数据到-tidb)。
+
+1. 创建配置文件 `tidb-lightning.toml`，并根据你的集群信息，填写如下配置：
```
Comment (Oreoxmt) [inline get-started-with-tidb-lightning.md:65]:
```suggestion
1. 创建配置文件 `tidb-lightning.toml`，并根据你的集群信息填写如下配置：
```

File: get-started-with-tidb-lightning.md [hunk 11]
```diff
@@ -79,16 +95,14 @@ TiDB Lightning 的安装包位于 TiDB 离线工具包中。下载方式，请
     pd-addr = "172.16.31.3:2379"
     ```
 
-4. 配置合适的参数运行 `tidb-lightning`。如果直接在命令行中用 `nohup` 启动程序，可能会因为 SIGHUP 信号而退出，建议把 `nohup` 放到脚本里面，如：
-
-    {{< copyable "shell-regular" >}}
+2. 运行 `tidb-lightning`。如果直接在命令行中用 `nohup` 启动程序，可能会因为 SIGHUP 信号而退出，建议把 `nohup` 放到脚本里面，如：
```
Comment (Oreoxmt) [inline get-started-with-tidb-lightning.md:98]:
```suggestion
2. 运行 `tidb-lightning`。为避免直接在命令行使用 `nohup` 启动程序时因 `SIGHUP` 信号导致的程序退出，建议将 `nohup` 命令放入脚本中。示例如下：
```

**Review by Oreoxmt — APPROVED** [review]

**Comment by rpaik** [comment]
👋 @TinaZhuzhu,  just a friendly reminder to submit your PR for TiDB Docs Dash by January 12 at 07:59 UTC. 🙏

Please reach out if you have any questions or need any help.

**Pull request merged**